charts/cinder-csi-plugin/values.yaml

```diff
@@ -93,6 +93,11 @@ csi:
     nodeSelector: {}
     tolerations: []
 
+# Log verbosity level.
+# See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md
+# for description of individual verbosity levels.
+logVerbosityLevel: 0
```

Comment:
if nothing is passed in argument , isnt default log level would be 2?

Reply:
@ramineni do you have some reference for this where it says it really defaults to 2? If not I'll go check it myself later today or in the coming days.

Reply:
Because AFAIK it indeed defaults to zero:

> `-v=0`
>	Enable V-leveled logging at the specified level.

https://pkg.go.dev/github.com/kubernetes/klog

Reply:
But that's just what I implied when I saw `-v=0`. I couldn't find anything explicitly stating what's the default level though, so that's why I'm asking if you saw it somewhere else.

Reply:
Some more references:

> Whether an individual call to V generates a log record depends on the setting of the -v and --vmodule flags; both are off by default.

https://pkg.go.dev/github.com/kubernetes/klog#V

Reply:
@gman0 Its there in same link you have mentioned in comment above :)

Reply:
Yeah, I agree

Reply:
@ramineni I think https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md summarizes the recommendations and actual defaults that klog is configured with. Either way I'll set the level to 2 as we just agreed.